js/components/Table/Row.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { map } from 'lodash';
+import PropTypes from 'prop-types';
+
+class Row extends React.Component {
+  constructor() {
+    super();
+
+    this.createRow = this.createRow.bind(this);
+  }
+
+  createRow(data) {
+    const row = [
+      {
+        title: data.user.data.first_name,
+      },
+      {
+        title: data.quantity === null ? 0 : data.quantity,
+      },
+    ];
+
+    return row;
+  }
+
+  render() {
+    const content = this.createRow(this.props.data);
+
+    return (
+      <tr className="table__row">
+        {content.map((cell, index) => <td className="table__cell" key={index}>{cell.title}</td>)}
+      </tr>
+    );
+  }
+}
+
+Row.propTypes = {
+  data: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
```

Comment:
Sorry to be nit-picky again! but if we are gonna disable all the prop type rules, then we aren't really using it, so we can just remove the `propTypes` import up top and remove these lines so that we don't try to set validation rules

Reply:
don't be sorry - this is good to know! Thank you!